### PR TITLE
doc: Bluetooth software maturity level update

### PR DESCRIPTION
**Pull request by yuxCai**
nRF54L05 and nRF54L10 are added to the table

### CODE REVIEW

File: doc/nrf/releases_and_maturity/software_maturity.rst [hunk 1]
```diff
@@ -687,6 +717,8 @@ The following table indicates the software maturity levels of the support for ea
         - Supported
         - Experimental
         - Supported
+        - Supported
+        - Supported
```
Comment (divipillai) [inline doc/nrf/releases_and_maturity/software_maturity.rst:721]: Shouldn't we also cover the proprietary Bluetooth features (from line no:731).

**Review by richabp — COMMENTED** [review]
Protocol section needs an update as well.

**Review by divipillai — APPROVED** [review]

File: doc/nrf/releases_and_maturity/software_maturity.rst [hunk 2]
```diff
@@ -183,6 +183,8 @@ The following table indicates the software maturity levels of the support for ea
      - nRF52840
      - nRF5340
      - nRF54H20
+     - nRF54L05
```
Comment (seko-nordic) [inline doc/nrf/releases_and_maturity/software_maturity.rst:186]: General question: if we add a HW here, and mark a protocol as not supported, do we still need to add that HW to that protocol specific table and mark all protocol features as not supported? IMO that does not make sense, but I see we do it for some HW. Looks inconsistent now, we should clean it up, may be in separate PR.

**Review by richabp — APPROVED** [review]

File: doc/nrf/releases_and_maturity/software_maturity.rst [hunk 3]
```diff
@@ -202,6 +202,8 @@ The following table indicates the software maturity levels of the support for ea
      - --
      - --
```
Comment (divipillai) [inline doc/nrf/releases_and_maturity/software_maturity.rst:203]: These entries could also be removed. This will also fix the doc build error. This is with the understanding that Bluetooth has support for nRF54L05 and L10.

Comment (yuxCai) [reply doc/nrf/releases_and_maturity/software_maturity.rst]: now fixed

**Review by Tschet1 — APPROVED** [review]
